Tasks/AzureKeyVault/Strings/resources.resjson/zh-TW/resources.resjson

```diff
@@ -0,0 +1,78 @@
+  "loc.input.help.enableDeploymentPrerequisitesForCreate": "啟用此選項即可使用自我簽署憑證，透過連接埠 5986 上的 HTTPS 通訊協定來設定 Windows 遠端管理 (WinRM) 接聽程式。必須要有此設定，才能在 Azure 機器上執行部署作業。若目標虛擬機器受 Load Balancer 支援，請確認已設定目標連接埠 (5986) 的輸入 NAT 規則。",
+  "loc.input.label.enableDeploymentPrerequisitesForSelect": "啟用部署必要條件",
+  "loc.input.help.enableDeploymentPrerequisitesForSelect": "啟用此選項即可使用自我簽署憑證，透過連接埠 5986 上的 HTTPS 通訊協定來設定 Windows 遠端管理 (WinRM) 接聽程式。必須要有此設定，才能在 Azure 機器上執行部署作業。若目標虛擬機器受 Load Balancer 支援，請確認已設定目標連接埠 (5986) 的輸入 NAT 規則。",
+  "loc.input.label.outputVariable": "資源群組",
+  "loc.input.help.outputVariable": "提供資源群組之變數的名稱。您可將變數作為 $(variableName) 使用，以參考後續工作中的資源群組，例如在 PowerShell 中部署應用程式時，參考目標電腦工作。<br>僅當選取的動作為 Create、Update 或 Select 時，此選項才有效，而此選項在選取現有的資源群組時為必要選項。",
+  "loc.messages.ARG_CreatedResourceGroup": "[Azure Resource Manager]已建立資源群組 '{0}'",
+  "loc.messages.ARG_StartingMachine": "[Azure Resource Manager]正在啟動電腦 '{0}'",
+  "loc.messages.ARG_StartedMachine": "[Azure Resource Manager]已從 Azure 提供者啟動電腦 '{0}'",
+  "loc.messages.ARG_StoppingMachine": "[Azure Resource Manager]正在停止電腦 '{0}'",
+  "loc.messages.ARG_StoppedMachine": "'{0}' 來自 Azure 提供者",
+  "loc.messages.ARG_DeletingMachine": "[Azure Resource Manager]正在刪除電腦 '{0}'",
+  "loc.messages.ARG_DeletedMachine": "[Azure Resource Manager]已從 Azure 提供者刪除電腦 '{0}'",
+  "loc.messages.ARG_DeletingResourceGroup": "[Azure Resource Manager]正在刪除資源群組 '{0}'",
+  "loc.messages.ARG_DeletedResourceGroup": "[Azure Resource Manager]已刪除資源群組 '{0}'",
+  "loc.messages.ARG_EnsureResourceGroupWithMachine": "請確定資源群組 '{0}' 存在，且其中至少具有一部虛擬機器",
+  "loc.messages.ARG_GettingVmStatus": "[Azure 呼叫]正在取得 VM '{0}' 的狀態",
+  "loc.messages.ARG_GotVmStatus": "[Azure 呼叫]已取得 VM '{0}' 的狀態",
+  "loc.messages.ARG_GettingExtensionStatus": "[Azure 呼叫]正在取得 VM '{1}' 的自訂指令碼擴充功能 '{0}'",
+  "loc.messages.ARG_GotExtensionStatus": "[Azure 呼叫]已取得 VM '{1}' 的自訂指令碼擴充功能 '{0}'",
+  "loc.messages.ARG_SettingExtension": "[Azure 呼叫]正在設定 VM '{1}' 的自訂指令碼擴充功能 '{0}'",
+  "loc.messages.ARG_SetExtension": "[Azure 呼叫]設定 VM '{1}' 的自訂指令碼擴充功能 '{0}'",
+  "loc.messages.ARG_RemovingExtension": "[Azure 呼叫]正在移除 VM '{1}' 的自訂指令碼擴充功能 '{0}'",
+  "loc.messages.ARG_RemovedExtension": "[Azure 呼叫]已移除 VM '{1}' 的自訂指令碼擴充功能 '{0}'",
+  "loc.messages.ARG_NetworkInterfaceNotFound": "[Azure 呼叫]資源群組 {1} 下找不到虛擬機器識別碼為 {0} 的網路介面",
+  "loc.messages.ARG_EmptyRGName": "[Azure 呼叫]資源群組名稱及虛擬機器識別碼不應為 null 或空白",
+  "loc.messages.ARG_UseSpnAuth": "憑證式驗證只能搭配 [選取資源群組] 動作使用。請使用認證或服務主體名稱 (SPN) 驗證選取 Azure 訂用帳戶。",
+  "loc.messages.ARG_ProvideOutputVariable": "因為您已指定 [選取資源群組] 選項，所以請提供輸出變數名稱。",
+  "loc.messages.ARG_UnsupportedAzurePSVersion": "未安裝 Azure Powershell Cmdlet 要求的最低版本 {0}。您可以遵循下列網址中的指示取得最新 Azure powershell: {1}",
+  "loc.messages.ARG_AzureRMModuleNotFound": "未安裝必要的 AzureRM Powershell 模組。您可以遵循下列網址中的指示取得最新 Azure Powershell: {0}",
+  "loc.messages.ARG_DeploymentFailed": "資源群組部署 '{0}' 失敗",
+  "loc.messages.ARG_ValidationFailed": "資源群組部署範本驗證失敗，錯誤碼為 {0}，錯誤訊息為: {1}",
```

Comment:
We do not need to check-in other languages resources. Hence deleted these files.